tutorials/preprocessing/plot_60_maxwell_filtering_sss.py

```diff
@@ -106,18 +111,100 @@
 raw.info['bads'] = []
 raw_check = raw.copy()
 raw_check.pick_types(meg=True, exclude=()).load_data().filter(None, 40)
-auto_noisy_chs, auto_flat_chs = mne.preprocessing.find_bad_channels_maxwell(
+auto_noisy_chs, auto_flat_chs, auto_scores = find_bad_channels_maxwell(
     raw_check, cross_talk=crosstalk_file, calibration=fine_cal_file,
-    verbose=True)
+    return_scores=True, verbose=True)
 print(auto_noisy_chs)  # we should find them!
 print(auto_flat_chs)  # none for this dataset
-raw.info['bads'].extend(auto_noisy_chs + auto_flat_chs)
 
 ###############################################################################
-# But this algorithm is not perfect. For example, it misses ``MEG 2313``,
-# which has some flux jumps, because there are not enough flux jumps in the
-# recording. So it can still be useful to manually inspect and mark bad
-# channels:
+# Now we can update the list of bad channels in the dataset.
+
+bads = [*raw.info['bads'], *auto_noisy_chs, *auto_flat_chs]
```

Comment:
bads = raw.info['bads'] + auto_noisy_chs + auto_flat_chs

seems simpler especially for non python gurus